universal-application-tool-0.0.1/app/services/program/ProgramDefinition.java

```diff
@@ -47,12 +50,77 @@ public static Builder builder() {
   /** A human readable description of a Program, localized for each supported locale. */
   public abstract LocalizedStrings localizedDescription();
 
-  /** The list of {@link BlockDefinition}s that make up the program. */
   public abstract ImmutableList<BlockDefinition> blockDefinitions();
 
   /** The list of {@link ExportDefinition}s that make up the program. */
   public abstract ImmutableList<ExportDefinition> exportDefinitions();
 
+  /**
+   * Returns a program definition with block definitions such that each enumerator block is
```

Comment:
praise: I can't remember if this is what the comment said before we chatted, but I think the docstring is pretty dang good!